executor/insert_common.go

```diff
@@ -216,6 +240,13 @@ func (e *InsertValues) evalRow(cols []*table.Column, list []expression.Expressio
 
 		offset := cols[i].Offset
 		row[offset], hasValue[offset] = val1, true
+		if expr.Flag()&expression.FlagChunkReused > 0 {
```

Comment:
The `FlagChunkReused` makes me confused, if the expression have a reuse flag, than we create a new chunk ?

Reply:
yes, expression reuse chunk, so we can not reuse old chunk for following eval~

Reply:
How about `expression.HoldChunkMemory`, it the returned result of an expression need to hold the memory of the chunk, we need to allocate a new chunk.